pkg/agent/apiserver/handlers/ovsflows/handler.go

```diff
@@ -42,6 +50,26 @@ func getAllFlows(aq querier.AgentQuerier) ([]Response, error) {
 	return resps, nil
 }
 
+// nil is returned if the flow table can not be found (the passed table name or
+// number is invalid).
+func getTableFlows(aq querier.AgentQuerier, table string) ([]Response, error) {
```

Comment:
I think it's a bit nasty that `table` can either be the string name or the table id. I'd recommend having 2 functions instead:

Reply:
Do not get what you mean - the table name or number is from a single parameter in the API request and a single actctl command argument. Do you mean you want to use two parameters/arguments?

Reply:
Yes I meant having 2 parameters in the API (`tableName` and `tableID`), but maybe it makes things more complicated? I'm not sure if this is an actual red flag in terms of API design.

Reply:
I see. I think for antctl better to use one argument which is more convenient for users. For API, maybe you are right, but now the antctl framework is not very convenient to do argument validation and changes before calling API.